modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/MyWorkflowTasks.macro

```diff
@@ -81,7 +83,9 @@ definition {
 				locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE",
 				value1 = "${displayDateMonth}/${futureDay}/${displayDateYear}");
 
-			Type(
+			var startDateTime = StringUtil.removeSpaces(${variavelDoTime});
+
+			Type.sendKeys(
```

Comment:
The behavior has changed. It no longer has a custom html to specify the click. It is necessary to use this function.
All the tests that use this macro have been tested and are in the ticket description